### DISCUSSION TITLE
Problem with LedCarveTask

### DISCUSSION THREAD

**Discussion by vitorlco** [sepinf-inc/IPED#1804] [General]
We successfully imported  LED 1.3.0 hashes into the iped Hash Database usuing version>=4.0.0. When running iped, I successfully configure the hashes, however, an error message is displayed:
[ERROR] [task.carver.LedCarveTask] LED hashes must be loaded into IPED hashes database to enable LedCarveTask.

**Comment by wladimirleite** [comment]
> We successfully imported LED 1.3.0 hashes into the iped Hash Database using version>=4.0.0. 

Hello @vitorlco! 
I guess you meant LED 1.30, not 1.3.0, right?
Besides that, how did you import LED hashes into IPED hashes database? 

The error message you are getting means there is no record in the proper format in your database.
LedCarveTask needs information in a specific format (e.g. hashes of the first 512 bytes and 64 kB, file length etc.).
The exact expected CSV format (and a sample data line):
```text
MD5,SHA1,EDONKEY,SHA256,status,set,md5_512,md5_64k,fileLength,fileExt,ledGroup,ledCategory
000001212D1A0312FF896AE308458412,5C187220CAC8D599FBEDF2D6196C36CFE2AE1E39,B531A964548EE4F4322257FBC1A5D25B,492C49FD7526DAFB9DCF327F887AAAE633CF572B1208B0BC5B029D960702308E,pedo,LED,E0695993EBD0BC0E210071A348C66484,422F10EE9CA4D17DF912FFDE879C62F4,250443,jpg,laudo-0069-2017-setec-sp,1
```

If you have a set of text files with LED hashes (LED version >= 1.29), you can convert them to a single CSV using the program posted in another discussion https://github.com/sepinf-inc/IPED/discussions/706#discussioncomment-4022651. And then import this CSV to IPED hashes database.
